iron-collapse.html

```diff
@@ -199,9 +199,9 @@
         }
         // Go to startSize without animation.
         this.style[this._dimensionMax] = startSize;
-        // Force layout to ensure transition will go. Set offsetHeight to itself
+        // Force layout to ensure transition will go. Set scrollTop to itself
         // so that compilers won't remove it.
-        this.offsetHeight = this.offsetHeight;
+        this.scrollTop = this.scrollTop;
```

Comment:
Why is scrollTop different than offsetHeight? There's nothing in the description to explain this :)